### PR TITLE
Add watchlists etc. in backend

### PR DESCRIPTION
**Pull request by Willenbrink**

### CODE REVIEW

**Review by Blezie — COMMENTED** [review]
It looks good.

**Comment by Willenbrink** [comment]
Sorry, blocking the merge right now. I will push persistence to the branch in 5 min or so :)

**Review by petrusmatiros — APPROVED** [review]
Looks good! Approved

**Pull request merged**
